awx/main/management/commands/inventory_import.py

```diff
@@ -544,14 +561,15 @@ def _delete_group_children_and_hosts(self):
             for offset in range(0, len(del_child_group_pks), self._batch_size):
                 child_group_pks = del_child_group_pks[offset:(offset + self._batch_size)]
                 for db_child in db_children.filter(pk__in=child_group_pks):
-                    group_group_count += 1
+                    group_group_removals += 1
```

Comment:
Note that `group_group_count` meant the number of group-children relationships _removed_.